### PR TITLE
Moved kotsadm-kots error to correct column

### PR DESCRIPTION
**Pull request by catherinepope**
# Description
Moved kotsadm-kots error to correct column

### CODE REVIEW

File: jekyll/_cci2/server-3-overview.adoc [hunk 1]
```diff
@@ -238,8 +238,9 @@ but can also be configured to be exported to other places (i.e. Datadog or Prome
 
 | kotsadm-kots
 | Licensing
-| The main KOTS application. Runs the KOTS Admin Console where upgrades and configuration of server take place	No admin console available.
+| The main KOTS application. Runs the KOTS Admin Console where upgrades and configuration of server take place.
 | No upgrades or configuration possible for server.
+No admin console available.
```
Comment (rosieyohannan) [inline jekyll/_cci2/server-3-overview.adoc:243]:
```suggestion
| No upgrades or configuration possible for server. No admin console available.
```

Comment (rosieyohannan) [reply jekyll/_cci2/server-3-overview.adoc:243]: I think this should be in the "what happens if it fails" column?

Comment (catherinepope) [reply jekyll/_cci2/server-3-overview.adoc:243]: Doesn't that shift it into the Notes column? This is what my version looks like for me in Docker:
kotsadm

Comment (rosieyohannan) [reply jekyll/_cci2/server-3-overview.adoc:243]: oh ok cool, the pic looks right so let's go with that, sorry!

Comment (rosieyohannan) [reply jekyll/_cci2/server-3-overview.adoc:243]: merge away!

Comment (catherinepope) [reply jekyll/_cci2/server-3-overview.adoc:243]: Those tables are really eye-boggling! 👀

**Pull request merged**